internal/config/cassandra.go

```diff
@@ -18,12 +18,57 @@
 package config
 
 type Cassandra struct {
-	Addrs []string `json:"addrs" yaml:"addrs"`
+	Hosts                  []string `json:"hosts" yaml:"hosts"`
+	CQLVersion             string   `json:"cql_version" yaml:"cql_version"`
+	Timeout                string   `json:"timeout" yaml:"timeout"`
+	ConnectTimeout         string   `json:"connect_timeout" yaml:"connect_timeout"`
+	Port                   int      `json:"port" yaml:"port"`
+	NumConns               int      `json:"num_conns" yaml:"num_conns"`
+	Consistency            string   `json:"consistency" yaml:"consistency"`
+	MaxPreparedStmts       int      `json:"max_prepared_stmts" yaml:"max_prepared_stmts"`
+	MaxRoutingKeyInfo      int      `json:"max_routing_key_info" yaml:"max_routing_key_info"`
+	PageSize               int      `json:"page_size" yaml:"page_size"`
+	DefaultTimestamp       bool     `json:"default_timestamp" yaml:"default_timestamp"`
+	MaxWaitSchemaAgreement string   `json:"max_wait_schema_agreement" yaml:"max_wait_schema_agreement"`
+	ReconnectInterval      string   `json:"reconnect_interval" yaml:"reconnect_interval"`
+	// ConvictionPolicy:       &SimpleConvictionPolicy{},
+	ReconnectionPolicy    *ReconnectionPolicy `json:"reconnection_policy" yaml:"reconnection_policy"`
+	WriteCoalesceWaitTime string              `json:"write_coalesce_wait_time" yaml:"write_coalesce_wait_time"`
+
+	Keyspace string `json:"keyspace" yaml:"keyspace"`
+	KVTable  string `json:"kv_table" yaml:"kv_table"`
```

Comment:
table? keyspace?

Reply:
GoCQL cannot connect to multiple keyspaces simultaneously. Then, we decided to hold multiple tables in single keyspace. :cry:

Reply:
omg